lib/Installation/NtpConfiguration/NtpConfigurationPage.pm

```diff
@@ -0,0 +1,42 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: The module provides interface to act on the
+#          ntp Configuration page.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+package Installation::NtpConfiguration::NtpConfigurationPage;
+use parent 'Installation::Navigation::NavigationBase';
+use strict;
+use warnings;
+
+sub init {
+    my $self = shift;
+    $self->SUPER::init();
+    $self->{txb_ntp_server} = $self->{app}->textbox({id => '"Y2Caasp::Widgets::NtpServer"'});
+    return $self;
+}
+
+sub is_shown {
+    my ($self) = @_;
+    return $self->{txb_ntp_server}->exist();
+}
+
+sub get_ntp_config {
+    my ($self) = @_;
+    return $self->{txb_ntp_server}->value();
+}
+
+sub get_ntp_configuration_page {
```

Comment:
I don't see this function being used anywhere.

Reply:
Need it to check if we get the expected page.  Added to use this function at 'tests/installation/ntp_configuration.pm'

Reply:
you need to run it chained in line 39.

Reply:
Before get the ntp server value, I need make sure the NTP page show up.
So run it chained in line 28 instead of 39.

Reply:
you need to chain the call in both places, if you check existing libyui-rest-api call you will see this strategy all  over the source code.